ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/SqlLedgerReaderWriter.scala

```diff
@@ -35,7 +36,7 @@ import com.daml.platform.common.MismatchException
 import com.daml.resources.{Resource, ResourceOwner}
 import com.google.protobuf.ByteString
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.{ExecutionContext => ScalaExecutionContext, Future => ScalaFuture}
```

Comment:
I do encourage in the documentation renaming `scala.concurrent` itself and using the containing package as a qualifier in all references rather than renaming individual symbols, but you do you.

Reply:
Yeah, I think I'd like to be consistent there, so I'll change it.